sci-visualization/fityk/fityk-1.2.9.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/sci-visualization/fityk/fityk-1.2.1.ebuild,v 1.3 2014/11/28 21:37:36 pacho Exp $
+
+EAPI=5
+
+WX_GTK_VER="3.0"
+PYTHON_COMPAT=( python{2_6,2_7,3_2,3_3,3_4} )
+
+inherit autotools-utils fdo-mime python-r1 wxwidgets
+
+DESCRIPTION="General-purpose nonlinear curve fitting and data analysis"
+HOMEPAGE="http://fityk.nieto.pl/"
+SRC_URI="https://github.com/wojdyr/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="gnuplot nlopt readline python static-libs wxwidgets"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+CDEPEND="
+	>=sci-libs/xylib-1
+	>=dev-lang/lua-5.1
+	nlopt? ( sci-libs/nlopt )
+	python? ( ${PYTHON_DEPS} )
+	readline? ( sys-libs/readline )
```

Comment:
Unfortunately, repoman didn't. Chose `:*`, as the (configure.ac)[https://github.com/wojdyr/fityk/blob/master/configure.ac] tells that libreadline > 4.2 is needed.